packages/abi-typegen/test/fixtures/forc-projects/index.ts

```diff
@@ -30,12 +32,20 @@ export enum AbiTypegenProjectsEnum {
   VECTOR_SIMPLE = 'vector-simple',
 }
 
-export const getTypegenForcProject = (
+export const getTypegenForcProject = <T extends boolean = false>(
   project: AbiTypegenProjectsEnum,
-  build: 'release' | 'debug' = 'release'
-) =>
-  getForcProject<JsonAbi>({
+  transpile: T = false as T
```

Comment:
```suggestion
  { transpile: T = false as T }
```

NIT - it's usually better to have named options for increased readability everywhere else:

```ts
const project = getTypegenForcProject(.., { transpile: true });
```

Instead of:

```ts
const project = getTypegenForcProject(.., true);
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2862/commits/6b8edc923056061e1161ff7af5e7db6f3bfaffbb